src/routes/MapExplorer/MapExplorer.js

```diff
@@ -181,8 +187,8 @@ export default class MapExplorer extends React.Component {
   }
 
   toggleDrawer = (e) => {
-    this.setState({ showDrawer: !this.state.showDrawer })
-    this.props.toggleDrawer(!this.state.showDrawer)
+    this.setState({ hideDrawer: !this.state.hideDrawer })
+    this.props.toggleDrawer(!this.state.hideDrawer)
```

Comment:
I added context here per my comment above

```
toggleDrawer = (e) => {
    this.setState({ hideDrawer: !this.state.hideDrawer })
    let context
    if (isAllCommunitiesPath(this.props.location.pathname)) context = 'all'
    else if (isPublicPath(this.props.location.pathname)) context = 'public'
    this.props.toggleDrawer(!this.state.hideDrawer, context)
 }
```

And imported these at top 

```
import { isAllCommunitiesPath, isPublicPath } from 'util/navigation'
```

Reply:
that seems pretty ugly, it should be able to figure this out on its own using the routeParams

Reply:
and like i said, its working for me, what exact steps are you doing?

Reply:
yes agreed it's not the prettiest... I'm clicking the icon in the top left, selecting "Public Communities & Posts" or "All My Communities", selecting the map, and clicking the x. 

...but hmm... now it is working. not sure what happened before. 🤔 you can disregard this!